_pages/cfp.md

```diff
@@ -59,10 +48,10 @@ We ask for three things other than your basic information: An abstract, a timeli
 **Do not include any personally identifying information in your abstract, timeline or answer to the short answer question. This includes:**
```

Comment:
It doesn't automatically disqualify you. If it's just a website or similar that gets left in we can just redact it. Otherwise it's at our discretion which is crappy. 

I'll add a line like "including personally identifiable information in your submission may result in disqualification if the Speakers Team is unable to anonymize is sufficiently". Does that work?

Reply:
Actually would it be helpful to link to the Canadian Privacy Commissioner's definition of PII? https://www.priv.gc.ca/en/privacy-topics/privacy-laws-in-canada/the-personal-information-protection-and-electronic-documents-act-pipeda/pipeda_brief/#_what_is

Reply:
up to you but i think what we have now is pretty clear 🤷‍♀️

Reply:
adding "including personally identifiable information in your submission may result in disqualification if the Speakers Team is unable to anonymize is sufficiently" sounds a lot better to me